### ISSUE TITLE
Install on a Synology NAS but i don't find the download file

### ISSUE THREAD

**Issue by OzZy02100** [rogerfar/rdt-client#75]
I've install  rdt-client to a synology NAS, all seems great. I see the download file in the Real Debrid Client / Torrents.
But i can't find the file in my computer...

**Comment by rogerfar** [comment]
Did you install it as a docker container? If so, did you follow the Docker setup guide?
https://github.com/rogerfar/rdt-client/blob/master/README-DOCKER.md

**Comment by OzZy02100** [comment]
Yeah seems good now except that I have some issue. I'm stuck to this step "Torrent waiting for file selection".

**Comment by rogerfar** [comment]
Please send the full log with Debug settings when this issue occurs.
